### PR TITLE
Debug and time

### PR DESCRIPTION
**Pull request by magsteen**

### CODE REVIEW

File: ALT.java [hunk 1]
```diff
@@ -105,15 +74,18 @@ public int search(Graph graph, int startNumber, int endNumber, int[] landmarkNum
...
+                            //System.out.println(estimate);
+                            toNode.setEstimatedDistance(estimate);
+                        }
                         queue.add(toNode);
                         toNode.setEnqueued(true);
                     }
                 }
                 //System.out.println(!toNode.isEnqueued());
                 if (!toNode.isVisited() && !toNode.isEnqueued()) {
+                    System.out.println("GEH - ALT");
```
Comment (ngorke) [inline ALT.java]: sus

File: ALTPreprocessor.java [hunk 2]
```diff
@@ -37,33 +41,100 @@ private void fillArrayWithDistanceData(Graph graph, int[][] array) {
...
+        try {
+            cached = readCached(fileName);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        } catch (NoSuchAlgorithmException e) {
+            throw new RuntimeException(e);
+        }
+        if (cached) {
+            System.out.println("We cached the lil' homie!");
+        }
```
Comment (ngorke) [inline ALTPreprocessor.java]: Remove or change this print

File: Dijkstra.java [hunk 3]
```diff
@@ -55,6 +47,7 @@ public int search(int startNodeNumber, int endNodeNumber) {
                 }
 
                 if (!toNode.isVisited() && !toNode.isEnqueued()) { //Currently doesn't get used at all
+                    System.out.println("GEH - Dijkstra");
```
Comment (ngorke) [inline Dijkstra.java]: remove this one too

File: GraphReader.java [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+            int numberOfNodes = readNextInt(stream);
+            Node[] nodes = new Node[numberOfNodes];
+            Node[] invertedNodes = nodes.clone();
+    
+            int number;
+            double latt;
+            double longt;
+    
+            for (int i = 0; i < numberOfNodes; i++) {
+                if (i % 1200000 == 0 || i > 6892650) System.out.println("~ parsing nodes ~ : " + (i+1));
```
Comment (ngorke) [inline GraphReader.java]: Random numbers? :thinking:

File: GraphReader.java [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+            int fromNodeNumber;
+            int toNodeNumber;
+            int drivetime;
+            int length;
+            int speedlimit;
+            Edge edge;
+            Edge invertedEdge;
+    
+            for (int i = 0; i < numberOfEdges; i++) {
+                if (i % 3000000 == 0 || i > 15494450) System.out.println("~ parsing edges ~ : " + (i+1));
```
Comment (ngorke) [inline GraphReader.java]: Random numbers? :thinking:

**Review by ngorke — CHANGES_REQUESTED** [review]

File: ALT.java (hunk 1 above)
Comment (magsteen) [reply ALT.java]: yes

**Review by ngorke — APPROVED** [review]
Nice!

**Pull request merged**